Sources/App/Services/Helper/FileService.swift

```diff
@@ -10,19 +10,29 @@ import Foundation
 import Vapor
 
 
+/// Service for managing file paths
 enum FileService {
```

Comment:
I think most of this can move into the `QuestionnaireStorageService` and some of the other things can probably move into the constants definitions?